recipes/radioactivedecay/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "radioactivedecay" %}
+{% set version = "0.3.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radioactivedecay-{{ version }}.tar.gz
+  sha256: ddb0a1054cf659db465a7355be3f8ac203e81206aa90b07798ac0eeafb65d99a
+
+build:
+  number: 0
+  skip: true   # [py<36]
```

This looks like a pure Python package. You can just change this to:
```suggestion
  noarch: python
```

---

Done thanks